Sources/WebLinking.swift

```diff
@@ -76,30 +76,30 @@ extension Link {
     }
 
     self.uri = trim("<", ">")(uri)
-    self.parameters = parameters.reduce([:], combine: +)
+    self.parameters = parameters.reduce([:], +)
   }
 }
 
 /*** Parses a Web Linking (RFC5988) header into an array of Links
 - parameter header: RFC5988 link header. For example `<?page=3>; rel=\"next\", <?page=1>; rel=\"prev\"`
 :return: An array of Links
 */
-public func parseLinkHeader(header: String) -> [Link] {
+public func parseLinkHeader(_ header: String) -> [Link] {
```

Comment:
This would be more in-line with the Swift API guidelines: https://swift.org/documentation/api-design-guidelines/#parameter-names